lilach-server/src/main/resources/org/cshaifasweng/winter/application.properties

```diff
@@ -2,7 +2,7 @@
 spring.jpa.show-sql=true
 # TODO: Change this once we have a proper DB initialization process
 spring.jpa.hibernate.ddl-auto=create-drop
-spring.datasource.url=jdbc:mysql://localhost:3306/lilach
+spring.datasource.url=jdbc:mysql://localhost:3306/lilach?serverTimezone=UTC
```

Comment:
That's what we did together. Since it works on my machine as well, let's keep it.